### DISCUSSION TITLE
Seperate cache for different wiki ids

### DISCUSSION THREAD

**Discussion by MusaabKh** [SmartDataAnalytics/OpenResearch#239] [Ideas]
The idea is to update the fromCache function to create the cache based on the wikiId given instead of storing all the cache calls in a singular directory to differentiate between wikis properly.

**Comment by tholzheim** [comment]
Currently when logging in to dblpConf with wiki A and the cache is already initialized from wiki B the cache of B is used for A.
The question is if we want to support different wikis simultaneously (e.g. for test purposes) or not than we could hard code the behavior.

**Comment by WolfgangFahl** [comment]
I am in favor of being able to have different cache version and configure the cache via the REST API (Admin mode). That gives us a lot more options e.g. also trying to synchronize two different versions of a cache as we might need for the openresearch.org and openresearch clone content.
